backend/src/services/applicant/saveApplicant.ts

```diff
@@ -91,12 +92,29 @@ export const saveApplicant = async (
   applicant.address = address;
 
   await AppDataSource.getRepository(Applicant).save(applicant);
+
+  // send email to ourselves
   sendEmail({
     sender: config.email.sender,
     receiver: config.email.receiver,
-    subject: "Neuer Interresent",
-    text: "Ein neuer Interresent hat sich gemeldet.",
-    html: '<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Neuer Interresent</title></head><body><p>Ein neuer Interresent hat sich gemeldet.</p></body></html>',
+    subject: "Neuer Interessent",
+    paragraphs: ["Ein neuer Interessent hat sich gemeldet."],
+  });
+
+  // send email to applicant
+  const title = "Registrierung bestätigt";
+  const paragraphs = [
+    `Hallo ${request.address.firstname},`,
+    `danke, dass du dich beim ${appConfig.address.name} registriert hast. Wir werden deine Anmeldung zeitnah bearbeiten und du erhältst eine weitere Bestätigungsmail mit deinem Benutzernamen.`,
+    "Viele Grüße",
+    "Die ehrenamtlichen Mitglieder der Solawi Gemüseanbau in Graupa",
+  ];
+  sendEmail({
+    sender: config.email.sender,
+    receiver: request.address.email,
+    subject: title,
+    paragraphs,
   });
+
   ctx.status = http.created;
```

Comment:
Evtl. überlegen, ob man vorher die E-Mail überprüft, wie man es sonst so kennt

Reply:
Das Format der Adresse wird bereits im Frontend gecheckt, daher ist es denke ich hier nicht nochmal nötig.

Reply:
Allgemein ist es immer wichtig auch im Backend zu prüfen, weil Angreifer gehen nicht über das Frontend. Aber ich meinte nicht das E-Mail-Adressenformat, sondern dass die E-Mail-Adresse von dem Nutzer auch kontrolliert wird. Man kann ja jede x-beliebige E-Mail-Adresse angeben. Aber das ist deutlich komplexer, weil man dann sich in der Datenbank noch merken muss, dass der Nutzer den Bestätigungslink klicken muss.

Reply:
ja, sowas wäre definitiv schön. In der Kürze der Zeit schaffe ich das allerdings nicht (bin ab Montag für 2 Wochen weg). Daher wollte ich zumindest eine Minimallösung umsetzen. Auch vor dem Hintergrund, dass ich das Risiko für einen derartigen Angriff relativ gering sehe. Unsere Datenbank könnte man ja theoretisch jetzt schon mit Registrierungsanfragen zuspammen. Es ist aber so, dass, bevor sich der Nutzer wirklich anmelden kann, ein Admin manuell die Registrierung bestätigen muss.